system/core/HyphaRequest.php

```diff
@@ -151,6 +161,10 @@ public function getPort() {
 			return $_SERVER['SERVER_PORT'];
 		}
 
+		public function isSecure() {
+			return (array_key_exists('HTTPS', $_SERVER) && $_SERVER['HTTPS'] == 'on');
```

Comment:
r edundant parentheses (non blocking comment :-) )  maybe triple `===` ?

Reply:
I like extra parenthesis to emphasize what is being returned, though thinking about it, I can't really put down in words why it would actually help :-) I guess it's a habit that's more relevant when assigning an expression to a variable, not really needed for return. Triple === is a good idea.

Reply:
Fixed.